src/main/java/de/retest/recheck/cli/RecheckCli.java

```diff
@@ -1,22 +1,36 @@
 package de.retest.recheck.cli;
 
+import org.apache.commons.lang3.SystemUtils;
+import org.fusesource.jansi.AnsiConsole;
+
 import de.retest.recheck.cli.subcommands.Account;
 import de.retest.recheck.cli.subcommands.Commit;
 import de.retest.recheck.cli.subcommands.Completion;
-import de.retest.recheck.cli.subcommands.Show;
 import de.retest.recheck.cli.subcommands.Diff;
 import de.retest.recheck.cli.subcommands.Ignore;
+import de.retest.recheck.cli.subcommands.Show;
 import de.retest.recheck.cli.subcommands.Version;
 import picocli.CommandLine;
 import picocli.CommandLine.Command;
 import picocli.CommandLine.Option;
 
 // TODO Add Migrate command when https://github.com/retest/recheck.cli/issues/85#issuecomment-526102137 is addressed.
-@Command( name = "recheck", description = "Command-line interface for recheck.",
-		versionProvider = VersionProvider.class, subcommands = { Version.class, Show.class, Diff.class, Commit.class,
-				Ignore.class, Completion.class, CommandLine.HelpCommand.class, Account.class } )
-
+@Command( name = "recheck", //
+		versionProvider = VersionProvider.class, subcommands = { //
+				Account.class, //
+				CommandLine.HelpCommand.class, //
+				Commit.class, //
+				Completion.class, //
+				Diff.class, //
+				Ignore.class, //
+				Show.class, //
+				Version.class, //
+		} )
```

Comment:
I assume this is left over from the split. You should remove this or else this might produce merge conflicts